themes/default/data/registry/packages/aws-static-website.yaml

```diff
@@ -0,0 +1,14 @@
+category: Cloud
+component: true
+description: ""
+featured: false
+logo_url: ""
+name: aws-static-website
+native: true
+package_status: public_preview
+publisher: Pulumi
+repo_url: https://github.com/pulumi/pulumi-aws-iam
+schema_file_path: schema.yaml
+title: AWS Static Website
+updated_on: 1654121492
+version: v0.0.1
```

Comment:
This doesn't reflect the same version as in GitHub

Reply:
yeah, I know... I'm going to publish a 0.0.1 shortly.